op-chain-ops/genesis/config.go

```diff
@@ -158,5 +158,9 @@ func NewStorageConfig(hh *hardhat.Hardhat, config *DeployConfig, chain ethereum.
 		// TODO: this should be set to the MintManager
 		"_owner": common.Address{},
 	}
+	storage["ProxyAdmin"] = state.StorageValues{
+		// TODO: this needs to be configurable
```

Comment:
Do we want to set this config up now? Or is it fine if that's done in a different PR?

Reply:
I was going to do this in a different PR, was thinking of adding the following config values:

- `l1ProxyAdminOwner`
- `l2ProxyAdminOwner`

Then those values would be passed through to here